app/models/binary/explore/WebknossosZarrExplorer.scala

```diff
@@ -0,0 +1,49 @@
+package models.binary.explore
+
+import com.scalableminds.util.geometry.Vec3Double
+import com.scalableminds.util.tools.Fox
+import com.scalableminds.webknossos.datastore.dataformats.zarr.{ZarrDataLayer, ZarrLayer, ZarrSegmentationLayer}
+import com.scalableminds.webknossos.datastore.models.datasource.{Category, DataSource}
+
+import java.nio.file.Path
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class WebknossosZarrExplorer extends RemoteLayerExplorer {
+
+  override def name: String = "WEBKNOSSOS-based Zarr"
+
+  override def explore(remotePath: Path, credentialId: Option[String]): Fox[List[(ZarrLayer, Vec3Double)]] =
+    for {
+      dataSourcePropertiesPath <- Fox.successful(remotePath.resolve("datasource-properties.json"))
+      dataSource <- parseJsonFromPath[DataSource](dataSourcePropertiesPath)
+      ngffExplorer = new NgffExplorer
+      zarrLayers <- Fox.serialCombined(dataSource.dataLayers) {
+        case l: ZarrSegmentationLayer =>
+          for {
+            zarrLayers <- ngffExplorer.explore(remotePath.resolve(l.name), credentialId)
```

Comment:
I’d suggest renaming one of the zarrLayers variables. How about zarrLayersFromNgff for the inner or zarrLayersAdapted for the outer?